packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -294,11 +294,12 @@ const Updates = decorate([
             <CardBlock>
               <fieldset disabled={COMMUNITY}>
                 <p>
-                  {_('xoaBuild')} {state.xoaBuild} - {_('currentVersion')}{' '}
+                  {_('currentVersion')}{' '}
                   {defined(
                     () => state.installedPackages['xen-orchestra'],
                     'unknown'
                   )}{' '}
+                  - {_('xoaBuild')} {state.xoaBuild}{' '}
```

Comment:
Shouldn't we simply hide it instead of displaying unknown if the version couldn't be fetched?

Reply:
I agree it's better to display it as unknown to avoid users looking for it.